addons/website/static/src/scss/website.scss

```diff
@@ -361,7 +361,7 @@ body {
 // the selector is probably too specific and should rather be about extending
 // bootstrap if possible.
 #wrapwrap:not(.o_header_overlay) header, header.o_header_is_scrolled, header .o_navbar_mobile {
-    .nav-item > .nav-link > *, .nav-item > .nav-link::after, .js_language_selector span, .badge {
+    .nav-item > .nav-link > *, .nav-item > .nav-link::after, .js_language_selector span, .badge, .nav-item > .accordion-item * {
```

Comment:
This change is needed to make the text of the "submenu" elements take the color selected in the navbar options.
![image](https://github.com/user-attachments/assets/2eab2f43-0e3a-46a5-a045-d633498f20b8)

Reply:
Not a fan of this selector but as advertised by the comment above, the previous ones should be reviewed later at some point